main/src/main/java/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -759,7 +759,6 @@ public boolean onOptionsItemSelected(final MenuItem item) {
         } else if (menuItem == R.id.menu_export_persnotes) {
             new PersonalNoteExport().export(Collections.singletonList(cache), this);
         } else if (menuItem == R.id.menu_edit_fieldnote) {
-            ensureSaved();
```

Comment:
You added/removed calls to `ensureSaved()`. This seems unrelated to the task at hand. What is this supposed to do?

Reply:
This PR contains unrelated commits, see PR's commit history